var/spack/repos/builtin/packages/py-fenics-ffcx/package.py

```diff
@@ -12,24 +12,37 @@ class PyFenicsFfcx(PythonPackage):
     homepage = "https://github.com/FEniCS/ffcx"
     url = "https://github.com/FEniCS/ffcx/archive/v0.1.0.tar.gz"
     git = "https://github.com/FEniCS/ffcx.git"
-    maintainers = ["chrisrichardson", "garth-wells"]
+    maintainers = ["chrisrichardson", "garth-wells", "jhale"]
 
     version("main", branch="main")
-    version("0.5.0", sha256="3413409e5885e41e220f99e0f95cc817e94c4931143d1f700c6e0c5e1bfad1f6")
+    version(
+        "0.5.0.post0", sha256="039908c9998b51ba53e5deb3a97016062c262f0a4285218644304f7d3cd35882"
+    )
     version("0.4.2", sha256="3be6eef064d6ef907245db5b6cc15d4e603762e68b76e53e099935ca91ef1ee4")
-    version("0.3.0", sha256="33fa1a0cc5762f360033c25a99ec9462be933f8ba413279e35cd2c3b5c3e6096")
-    version("0.2.0", sha256="562a704163a6427e0341d267e69b8bf55d8f53d673829170b443191577e8d9b6")
-    version("0.1.0", sha256="98a47906146ac892fb4a358e04cbfd04066f12d0a4cdb505a6b08ff0b1a17e89")
+    version(
+        "0.3.0",
+        sha256="33fa1a0cc5762f360033c25a99ec9462be933f8ba413279e35cd2c3b5c3e6096",
+        deprecated=True,
+    )
+    version(
+        "0.2.0",
+        sha256="562a704163a6427e0341d267e69b8bf55d8f53d673829170b443191577e8d9b6",
```

Comment:
This also does not match the output from `spack checksum`, which says it should be:

```suggestion
        sha256="f1dcd0973980706aba145274aeddeb771d2d730efcdb9b4be10adbe964f40d90",
```